CMakePresets.json

```diff
@@ -39,15 +39,17 @@
             "name": "windows",
             "hidden": true,
             "cacheVariables": {
-                "VCPKG_BUILD_TLS12_DOWNLOADER": true
+                "VCPKG_BUILD_TLS12_DOWNLOADER": true,
+                "CMAKE_CXX_COMPILER": "cl.exe", "CMAKE_C_COMPILER": "cl.exe"
```

Comment:
These seem clearly broken....

Reply:
I'm not sure what you mean.

I can leave all these changes out if you prefer. In our experience building TTD with MSVC and clang, this eases out the VSCode-as-GUI experience, but it doesn't need to be part of our change.

I probably should split the properties into lines if this is to stay. That's my bad, and I'll fix.

Reply:
Talked offline. Removed.